### DISCUSSION TITLE
Meeting: March 26th, 2024 agenda

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#94] [Announcements]
**Time:** 5pm GMT / 9am Pacific
**Zoom:** [https://zoom.us/j/975841675](https://zoom.us/j/975841675?pwd=SUh4MjRLaEFKNlI3RElpWTdhRDVVUT09), dial-in passcode: 763054

Participants must abide by our [Code-of-Conduct](https://github.com/OAI/OpenAPI-Specification/blob/main/CODE_OF_CONDUCT.md#code-of-conduct).

![image](https://github.com/OAI/sig-moonwalk/assets/54582/09ac7e12-59cd-4061-a7ec-1e36bb6387a7)

| Topic | Owner | Decision/NextStep |
|-------|---------|---------|  
| | | |
Intros and governance meta-topics (5 mins) | SIG members | -- |
General discussion topics (list below) | SIG members | -- |
Can we make links successful? | SIG members | |
Identity location ADR (add link) | SIG members | |

**Comment by earth2marsh** [comment]
As discussed in the 2024-03-19 meeting, @hudlow's topic on the intersection of resource-oriented and schema model variations (e.g., the interaction of read-only, write-only, auto-create-then-read-only, etc. with different operations).  It is at the intersection of:

* #30 
* #56 

Also carrying over HTTP version support advertising (just the version, not any other details discussed) might be simple enough to move to an ADR given the use case details provided here:

* https://github.com/OAI/sig-moonwalk/discussions/45#discussioncomment-8791212

**Comment by kevinswiber** [comment]
Pointing out the obvious... an image of "OAI Meeting Accessibility" isn't particularly accessible.  Even the `alt` tag is just `"image"`.

**Reply by earth2marsh** [reply]
Good point! This was cribbed from the TDC weekly, so let's get some better alt text from the original slide and use at least some of that here and there? @ncaidin likely has the original.

**Comment by earth2marsh** [comment]
#95 is the tracking issue for setting up a Moonwalk Contributors team. The team has been created, but we should determine the rest of its members and their IDs before we call for assistance from the org owners.

**Comment by earth2marsh** [comment]
Apologies, but neither @darrelmiller nor I can make it today. Please carry on without us, just capture any key points in this discussion, if you're able.

Here are the action items from last week: https://github.com/OAI/sig-moonwalk/discussions/88#discussioncomment-8844145

**Comment by earth2marsh** [comment]
[Video](https://zoom.us/rec/play/_4XCUWPfWNej2JM8EeL1NKii428752QLv1npmL9Kn0yZXLCg6A3Sx6Of7F0RtJFaIYh80BaL56hvZ7Ts.85l3V3qoFU7T_LDW?canPlayFromShare=true&from=share_recording_detail&continueMode=true&componentName=rec-play&originRequestUrl=https%3A%2F%2Fzoom.us%2Frec%2Fshare%2FcnPcjsLChAHCf921Dwn35CGRq1hRZZWvVJRAoF6JvUv62uBeF7ZYLv65w7iH-WlS.SIbBKTPHnnLCiOU9) now available (suggest jumping to 14:17 where the technical bits start). Thanks for stepping in to host, @miqui!

**Reply by earth2marsh** [reply]
I wasn't able to attend today, so if there are topics to bring over to the next discussion, please add them to #96 .
